DLaB.ModelBuilderExtensions/PacModelBuilderCodeGenHack.cs

```diff
@@ -58,7 +58,7 @@ public PacModelBuilderCodeGenHack(DLaBModelBuilderSettings settings, ICodeGenera
                     FilesWritten.Add(filePath, keyValuePair.Value);
                 }
 
-                if (string.IsNullOrEmpty(str))
+                if (str == "##@")
```

Comment:
This isn't doing anything.

Reply:
`string str = string.IsNullOrWhiteSpace(Settings.ServiceContextName) ? "##@" : Settings.ServiceContextName;`
`str` will not be empty, isn't it? The warning message will never show.

What I understand is that the warning `ProxyTypesAssemblyAttribute not written` is to inform user that, when service context name is not set in setting, ProxyTypesAssemblyAttribute will not write to any file, so user have to write it manually, am I misunderstanding something?

Reply:
That code is what I got from decompiling the MS source code.  And yes, you are write in your understanding.  What I meant to say was that it won't change how the code will generate, since it is only a warning.

Reply:
Yeah, it won't change any generation logic but it will give a nice tip to user. You are preferring to keep as is?
And I have met same issue of duplicate proxy header when using Visual studio power platform extensions....Now that make sense.

Reply:
You're getting the same issue without using any EBG code generation services?

Reply:
> You're getting the same issue without using any EBG code generation services?

I mean there are similar features provided from MS, like this [extension](https://marketplace.visualstudio.com/items?itemName=microsoft-IsvExpTools.PowerPlatformToolsVS2022)
It also able to create strong type for entities.
I remember when I was using this tool I encountered duplicate proxy header, but it was some times ago, I haven't check it lately.
I'm not clear about if they also using 'EBG code generation services' or not...